src/app/views/settings/Settings.tsx

```diff
@@ -0,0 +1,164 @@
+import {
+  ChoiceGroup, Dialog, DialogType, IconButton
```

Comment:
We can put each of the imports on a new line to ease readability.

Reply:
The linter we have always used defaults to this.

Reply:
Does it show a lint error when you do this? It doesn't for me.